Good heavens, look at the time! It's REMAP NORTH O' CLOCK!

FreeStylaLT opened the pull request:
Here we go agaiiiin.

Core goals this time around were:
* Remove / rework superfluous, awkward hallways (the one under the courtroom and the prison wing one)
* Consolidate Brig to allow for easier pathing
* Allow for more opportunities for escape for Perma & fix its species compatibility
* Make the hallway that's left feel more like a proper, central hallway by opening it up
* Make dorms less shit

Big Map-viewer summaries:

## BEFORE:
![img](https://cdn.discordapp.com/attachments/251859957949923328/480363193940574208/NORTH_BEFORE.png)

## AFTER:
![img](https://cdn.discordapp.com/attachments/251859957949923328/480363189830156308/NORTH_AFTER.png)

### DORMS:
![img](https://cdn.discordapp.com/attachments/251859957949923328/480363207089848330/NORTH_DORMS_NEW.png)

*Moved Cryo to a more isolated location, kept the pod count the same
*Moved the Clown & Mime offices to dorms
*Removed Chang's walls
*Added some other minor flavor things

As you can see, the main goal here was to streamline its layout and reshuffle things in a more practical and (hopefully) aesthetically appealing way. The clown & mime were also the couple of jobs that seemed to fit spawning in this area as well.

Along with this, I opened up the whole area to the Fore hallway, again, to streamline it and make the northern hallway feel far less cramped, this came at the cost of the aforementioned Cryo move.

### COURTROOM / IAA / MAGISTRATE:
![img](https://cdn.discordapp.com/attachments/251859957949923328/480363199284379658/NORTH_COURTROOM_NEW.png)

Instead of that god-awful hallway, there's now a dedicated little lobby area going into IAA, and a re-mapped courtroom that will never get any use, but looks nicer anyway. The whole point here was to essentially turn these three things into an inter-connected complex, which is what it always should have been.

### BRIG / PERMABRIG:
![img](https://cdn.discordapp.com/attachments/251859957949923328/480365855759138816/NORTH_BRIG_NEW.png)

The big one.

*Remapped the Lobby
*Remapped the prison wing to be left of the Lobby
*Remapped Perma - it can now support space-proof species
*Moved the Firing Range, toilets
*Moved the cells
*Added tiered cells

As you can see, half of it is how it's always been, while the other half is.. completely remapped.

This is because the Prison Wing never fit into the brig, or really anywhere, as it was a separate, long hallway with a bunch of crap wildly attached to it. This instead, again, consolidated the prison wing into a cohesive, integrated part of Brig, more or less as how it "should" have been two years ago.

Perma is probably the biggest change - it's less isolated now and slightly less secure, as now ground-based break-outs are easier - as you no longer need to have a space suit for those. 

The trays in front of windows also have their irrigation system turned on - meaning you can't break out to space all on your own.

Beyond that, as per Kyet's suggestion, I also added tiered cells. If you're a good boy, you can end up in a fancier cell where you can read a book or play arcade and, when the timer is up.. just walk out with your stuff on your own:

![img](https://i.imgur.com/zPmNutU.gifv)

The idea here is to encourage players to behave so they'd have a far more appealing convenience in prison than the shitlers, who can get the dungeon-tiered cells down below.

That about wraps it up, beyond a few pipe-related improvements, there isn't much else to show / talk about, check out the images for anything else I might have missed.

Spartan6 commented:
Free stop

ID404-NotFound commented:
there is no large area clear of pipes/wires to do wire art.

TDSSS commented:
Looks neat, the only thing I dislike is how on the old map there was a simple wall instead of an rwall between courtroom and brig proper, which is a pretty easy and remote way to get into brig, especially if you have tools/an emag for the doors. Now with the rework, you need to get through the magistrate's office and then an rwall.

You've effectively removed the third, least watched way into brig.

datlo commented:
This PR nerfs clown rushing the toolbelt and multitool at tool storage. No.

Jokes aside, I like it, although this makes breakin into security/perma from the courtroom... different.

Still like it. Would have to try it out.

TravellingMerchant commented:
The new showers are awful, completely useless to antags. Ditch the seating area and fix that?

FreeStylaLT commented:
Updated the main mapviewer pic - fixed various missing disposal pipes, buttons, minor aesthetic fixes like floor paint.

FreeStylaLT commented:
Updated main mapviewer pic - fixed various minor issues like missing disposal pipes, missing cables, etc, tweaked aesthetics like turf paint

Allfd commented:
I think the change to security perma/cells will be interesting to see.  Visitors to perma seem possible now, as do visitors to the cells.

Only question, is would it look better of the lethal injection closet were blocked by a windoor instead of a full airlock?

KasparoVy commented:
Fantastic. This is looking great! Even if only for the novelty of a new layout it'll be fun to have.

pinatacolada commented:
the door in front of the lethal injection locker sucks, it will try to close while someone is standing on it to use the locker. I'd ditch it or replace it with a windoor, but it's a redundant door anyway since the locker has access, I'd really just ditch it

TDSSS commented:
IMO the two maint accesses on the right are a bit too close together. I realize it's hard since on the left you have no maint but I'd really like it if there was a way to get into the left half of brig without going through the lobby area, like the previous access through the courtroom.

pinatacolada commented:
wait is there a cell less than in the live map?

FreeStylaLT commented:
@pinatacolada yes, one cell was removed. 6 seemed too many for practically and it's difficult to map 6 in consistently.

Newest update, bunch of small tweaks, not visible in any pics yet:

- Added an a security bot to Armory, Sargeant-at-Armsky:
![img](https://i.imgur.com/PfWaDBw.png)
It will patrol between the two navbeacons.

- Changed the exterior Armory cameras to be motion-sensitive
- Increased perma flasher range to 3 (used to be 2)
- Changed holopad placement, now you can project yourself to the entirety of brig and dorms without getting cut off.
- Fixed minor issues.

Rurik123 commented:
Four things I notice, that aren't exactly bad, but worth noting.

- First off, iirc, and please correct me if I am wrong, hydroponics trays can only be moved via a wrench. Thus perma breakouts via space can only be done with a wrench, which has been placed behind a window in the lobby leading to perma itself. Clever, though I wonder how long it will take for officers to nick it first.

- Escaping from processing for antags will be more difficult since it has two airlocks to get through two instead of one. An officer leaving two open is more unlikely than one etc. Of course if someone has tools and is uncuffed, then they'll get through two airlocks no problem.

-Escaping from the brig main hallway will be more difficult for the same reason as last, two airlocks to get through.

- Dorms bathrooms have less utility for running away, but a single false wall will fix that.

Overall sec will become more secure with this remap, and non perma prisoners will have a tough time unbuckling and running, compared to before. I'm not deeming that as bad, but it should be noted that should someone be caught without tools and brought to brig, escape from darting through open airlocks went from unlikely to near impossible.

Allfd commented:
Increase in security for .... security, is not necessarily bad.  When I spectate I always do so with antag-hud on as it helps me get a sense of what is happening faster.  I would say that at least half, but probably more of those in temporary cells as non-antags.  

Escape attempts as a non-antag take away significant security focus from the round.  Even if we don't want to bwoink anyone over that, having it be far more difficult to waste security officer time is probably a good thing.

TDSSS commented:
I really don't see the need for the bot in the armory. Keeping an eye on the armory is part of the job of the warden who should never leave brig. If he can't keep people from stealing stuff from the armory which is literally one door awaay from his office, then he deserves to have armory looted. And antags that go through the trouble of getting into space to try and loot it deserve it.

Spacemanspark commented:
Dunno if you're willing to, but we really should remove the sketchkin from the gamma armory, there's literally no reason for it to be in there.

Spartan6 commented:
@spacemanspark Kyet is gonna do a seperare PR after this one.

ID404-NotFound commented:
I thought the sketchkin was there for comedy. Everyone gears up, and then last guy comes in, "where's mine?" *Hands them smol pistol*

TDSSS commented:
> Also, securitron Sargeant wouldn't affect people who are not yet wanted even
Set it to arrest for unauthorized weapons any any attempt to pick up a gun gets you arrested and alerts all of sec to your presence. Set it to authorized unidentifiable people and anyone that takes their ID off before breaking in (generally sensible) gets robusted.

Spartan6 commented:
The securitron will be effective if set up correctly. If not, it’s up to the AI to set the intruder to arrest.

ID404-NotFound commented:
Part of the problem is what "difficulty level" are we aiming for? Have some things become so easy they are stale? How much of the playerbase is looking for challenge, and how much are still inexperienced and will feel alienated by high difficulty?

ID404-NotFound commented:
is pipes fixed from north to south? Supply not connected, though server might be still running not-fixed one.

TDSSS commented:
This area here looks a bit weird, with how there's this sharp little 'hole' in the form of the station. Maybe fill it up and put a small secret room there? With fun/useless item or two. Or remains on a chair, or whatever.
![untitled](https://user-images.githubusercontent.com/32099540/44500258-07b70c80-a688-11e8-9d0c-3293ce54458d.png)

taukausanake commented:
@TDSSS I like the idea of a special secret room there but it's so close to the armory that you can't really make a good reason to. That and the reinforced walls there so unless you're going to put something super special there

TDSSS commented:
I mean, the fluffy cuffs/ballgag are also behind reinforced walls, really hard to get to. Doesn't stop sec from going for it.

ID404-NotFound commented:
The fluffycuffs and ballgag are behind a normal wall, which always has a poster over it

TDSSS commented:
Ah, shows what I know. Hmm, I think telecoms has a secret room with a bananium coin, so there'd be precedent anyway? I suppose if secret room is out, could also just make armory a bit larger to fill the space too.

KasparoVy commented:
Part of the point of a secret room is to have it where few wou

TDSSS commented:
I also feel like part of the point of those secrets rooms is to be flavourful, to tell a story. Not necessarily to actually give you something very useful. Most secret rooms have something neat like a toy mech, or a coin and so on.

TDSSS commented:
Thought after today's round where we executed someone and then dragged his body to medbay: Could you add a box of body bags to the execution room (or maybe brigbay?)

taukausanake commented:
Do we have a guillotine mapped in? I know it's not hard to make but might be nice to have one along with blood splatters and a dead person missing a head

Calecute commented:
Your map has some atmos problems. It looks to me like there at least 3 separate air supply networks. 

This will result in most of the station being unable to be refilled with air, unless the atmos know or realize this and spend a considerable amount of time to fix it. It will also result in antags using pipes to have less options and get stuck more often. 

You can confirm this by checking pressure in air supply pipes in corridor near cargo, seeing they have pressure. Going north and checking air supply pipes near vault, seing they have no pressure. Connecting both pipes. And going to the brig main corridor, seeing it's pipes still have no pressure.

As far as I can tell the 3 networks are: medbay, bridge, cargo, eng and atmos, forming the Aft Port network, the one with actual air in it. The other one is arrivals, fore corridor and the things in it, eva, and it goes into sec lobby, the Fore Port network. And then there’s is the network containing most of brig, dorms, service area, starboard hallway, science and escape, the Fore Starboard Network. Unfortunately, I couldn't check extensively, and did so only inside the game, without a mapping tool that would make it more convenient, so I might be wrong, but it is a place to start.

I'd also recheck the scrubbers network just to be sure.

FreeStylaLT commented:
Big ass update that re-remaps Dorms slightly, remaps EVA / Gateway, adds an Office Supplies room, a decommissioned maint execution room w/ a guillotine, fixes all pipe issues (hopefully) and adds and renews various areas in dorms / brig with new [sprites. ](https://cdn.discordapp.com/attachments/266931663894872064/482832925562241024/unknown.png)

ID404-NotFound commented:
Can we get updated pictures?

FreeStylaLT commented:
@ID404-NotFound You did! All pics were updated along with the commit.

ID404-NotFound commented:
Ok, I see the pipe going from forms to EVA maint, but where does it connect back to the main atmos loop? The connection used to be from bridge to dorms/sec maint as seen in the before

joniroq commented:
Posting on Git even though I Discord'd you for posterity.

Couple issues were discovered when trying to use the Disposals mailing system to send crates to the Kitchen, Security and Science (Research). As well as the Location of the security disposals meant that often the package would be spit out into an inaccessible corner (Until the arcade machine in the public lobby is dismantled)

ID404-NotFound commented:
EVA maintenance APC?

Calecute commented:
I think Prision shuttle bay apc is not conected to the network

uzername commented:
Those closets with swimsuits look not in place in arcade room. Also, consider moving glass entrance door one tile to left, so that seats are not in traffic route.

Calecute commented:
Also prision shuttle bay atmos is not conected to the network it seems

FreeStylaLT commented:
All pics updated with the newest (although minor) changes

Calecute commented:
Looking at the pics atmos supply still seems disconected in prision shuttle bay. Scruber and power seems good thou.

KasparoVy commented:
Hey @FreeStylaLT, minor wiring issue I noticed during the last playtest-- still reflected in the updated screenshots. See below

Thanks
![cyberiad new north barmaint wire issue](https://user-images.githubusercontent.com/12377767/44961624-c6cebb80-af03-11e8-9ea2-c467a2dce29e.PNG)

FreeStylaLT commented:
Fixed!

I should really squash the commits..

miniusAreas commented:
@FreeStylaLT I have a thank you note for you, and since this helps me even more... Dual thankyous. Im a mapper for Sunset Station, and metabrig was a piece of shit no one wanted to touch besides Genpop. Dispite being banned from para a long time ago, I remembered the armory and how much nicer it is compared to metastations. You helped me a lot, and heres proof of it.
https://cdn.discordapp.com/attachments/484282183737933835/491380777934913537/unknown.png
I thank you with great pride, sir. You are the best.

Quantum-M commented:
I think that maintanance room connecting to the permabrig and the one window looking into permabrig, brig side, will just end up being a pain for Security.

The window since inmates can break it (such as with the cultivator, broken lightbulb, or a stool, I think).

The maintenance room since it's just one tile and a wall away to being directly connected to the station, making it easily for inmates to quickly get out if they get aid that way.

variableundefined commented:
This PR has been deconflicted and should be nearly merge-ready. I'll start test merging it again. However another maintainer should merge this in as I am technically a co-author and merging this would be equivalent to a test merge.

SamHPurp commented:
No fire alarm in court room

variableundefined commented:
I know of the scorched black tiles. WIll fix.

SamHPurp commented:
![image](https://user-images.githubusercontent.com/7415122/47213404-d2325500-d392-11e8-9d70-706272cf291d.png)

Decal ordering should be reviewed across the map. This instance you can see the neighbouring tiles are differing even. imo the white should be below the warning stripe.

SamHPurp commented:
Semi-related. Once #9810 merges, it'll be worth dropping the magboots, as they'll be available from the suit storage units.

dovydas12345 commented:
![capture](https://user-images.githubusercontent.com/37041561/47214507-d0728c80-d3a6-11e8-946c-5d681e6c166b.PNG)
Don't think thats supposed to be there.

KasparoVy commented:
:(

ID404-NotFound commented:
:(

Spacemanspark commented:
:(

Allfd commented:
F

AffectedArc07 commented:
Can I ask why?

FreeStylaLT commented:
@AffectedArc07 the map changes that got merged in since this one was made created some massive conflicts that are almost impossible to solve in a sane manner, so I essentially gave up on this branch.
